addon/styles/_lightbox.scss

```diff
@@ -68,6 +69,33 @@
       display: flex;
       justify-content: center;
       padding: 0.5em;
+      margin-top: -48px;
+      z-index: 1020;
+      @media (max-width: 375px) {
+        user-select: none;
+        display: flex;
+        justify-content: center;
+        padding: 0.5em;
+        margin-top: 0px;
+        z-index: 1020;
```

Comment:
No need to overwrite these values in the media query if you are not changing them
```suggestion
```